src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+
+header {
+  width: 100%;
+  flex-shrink: 0;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 10px 50px;
+}
+
+.header__logo {
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+  flex-shrink: 0;
+}
+
+.nav__list {
+  display: flex;
+  gap: 20px;
+  list-style: none;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: #000;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.nav__link:last-child {
+  margin-right: 50px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+@media (max-width: 1024px) {
+  header {
+    padding: 10px 50px;
+  }
+  .nav__link {
+    margin-right: 50px;
+  }
+}
+
+@media (max-width: 1200px) {
+  header {
+    max-width: 1200px;
+    margin: 0 auto;
+  }
```

Comment:
you don't need to use media queries in this task

Reply:
Not fixed